File: VB/HitTestingAxisTitle/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...

Namespace HitTestingAxisTitle
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub chartControl1_ObjectHotTracked(ByVal sender As Object, ByVal e As HotTrackEventArgs) Handles chartControl1.ObjectHotTracked
            If TypeOf e.AdditionalObject Is AxisTitle Then
                MessageBox.Show(e.AdditionalObject.GetType().ToString())
            End If
            If TypeOf e.AdditionalObject Is AxisLabelItem Then
                MessageBox.Show(CType(e.AdditionalObject, AxisLabelItem).AxisValue.ToString())
            End If
        End Sub
    End Class
End Namespace

File: CS/HitTestingAxisTitle/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...

namespace HitTestingAxisTitle {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void chartControl1_ObjectHotTracked(object sender, HotTrackEventArgs e) {
            if (e.AdditionalObject is AxisTitle) {
                MessageBox.Show(e.AdditionalObject.GetType().ToString());
            }
            if (e.AdditionalObject is AxisLabelItem) {
                MessageBox.Show(((AxisLabelItem)e.AdditionalObject).AxisValue.ToString());
            }
        }
    }
}